Xamarin.Forms.Platform.Android/Renderers/ShellSearchView.cs

```diff
@@ -237,9 +238,9 @@ protected virtual void LoadView(SearchHandler searchHandler)
 			linearLayout.Dispose();
 		}
 
-		void OnSearchHandlerFocusChange(object sender, FocusChangeEventArgs e, SearchHandler searchHandler)
+		void OnSearchHandlerFocusChange(object sender, FocusChangeEventArgs e)
 		{
-			searchHandler.SetValue(SearchHandler.IsFocusedPropertyKey, e.HasFocus);			
+			if(SearchHandler != null) SearchHandler.SetValue(SearchHandler.IsFocusedPropertyKey, e.HasFocus);
```

Comment:
I believe that you can do the `?.` thing... Otherwise I would suggest to break it in a new line, to follow the code style for this project (:
```suggestion
			       SearchHandler?.SetValue(SearchHandler.IsFocusedPropertyKey, e.HasFocus);
```

Reply:
Of course!